.github/workflows/beta.yml

```diff
@@ -18,16 +18,30 @@ jobs:
         with:
           node-version: "18.x"
 
+      - name: Check for existing release PR
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          existing_pr=$(gh pr list --state open --json title --jq '.[] | select(.title | contains("Version Packages"))')
```

Comment:
shouldn't it be something like `Version Packages (beta)` to not conflict with main releases?